### ISSUE TITLE
 first- and second-order  derivatives

### ISSUE THREAD

**Issue by weiweiWYW** [GODqinbin/DPBoost#3]
Hi @QinbinLi ,

In your paper, only first-order derivatives are used, but in LightGBM, second order derivatives are usually used as well. Does DPBoost only rely on first-order derivatives?

**Comment by QinbinLi** [comment]
Hi @weiweiWYW ,

DPBoost considers second-order derivative as 1. Note that in LightGBM the second-order derivative is a constant (i.e., 1) if adopting squared loss.

**Comment by weiweiWYW** [comment]
Hi @QinbinLi ,

I'm mainly dealing with binary classification.  In DPBoost, it is treated as regression, and then output is converted to a binary value.  But if I use binary cross entropy, does mean that I could just use one-order derivative to compute the G and V ?  which are shown in your paper.

**Comment by QinbinLi** [comment]
Hi @weiweiWYW ,

I think only using one-order derivative is an alternative approach. However, its accuracy may not be as good as using both one-order and second-order derivatives.

**Comment by weiweiWYW** [comment]
OK,  thank you for your reply
